querybook/server/lib/query_executor/executor_template/templates.py

```diff
@@ -34,6 +34,13 @@
     username=FormField(regex="\\w+"),
     password=FormField(hidden=True),
     impersonate=FormField(field_type=FormFieldType.Boolean),
+    proxy_user_identifier=FormField(
```

Comment:
will proxy_user_id or proxy_user_identifier be used as the param?

Reply:
My Bad. It would be `prox_user_id`. Made the change.